src/main/java/io/github/sefiraat/networks/slimefun/network/NetworkQuantumStorage.java

```diff
@@ -255,8 +282,28 @@ public int[] getSlotsAccessedByItemTransport(ItemTransportFlow flow) {
             @Override
             public void newInstance(@Nonnull BlockMenu menu, @Nonnull Block block) {
                 menu.addMenuClickHandler(ITEM_SET_SLOT, (p, slot, item, action) -> {
+                    final QuantumCache cache = CACHES.get(menu.getLocation());
                     if (action.isShiftClicked()) {
                         toggleVoid(menu);
+                    } else if (
+                        cache != null &&
+                        cache.getSupportsCustomMaxAmount() &&
+                        p.getItemOnCursor().getType() == Material.AIR
+                    ) {
+                        p.closeInventory();
+                        p.sendMessage(Theme.WARNING + "Type the new maximum storage size for this Quantum Storage.");
```

Comment:
I've added a little extra to say that it must be more than 0, but adding a max number I don't think is really needed

Reply:
The point of a max number is so a player doesn't enter some egregious number because the name has infinity in it

I said this because I've seen players be like, "why does it stop at >insert_max_int< it's supposed to be infinity!!!!???"